source/FreeRTOS_DHCPv6.c

```diff
@@ -379,7 +379,7 @@
 
         /* Check for clashes. */
 
-        vIPReloadDHCP_RATimer( ( struct xNetworkEndPoint * ) pxEndPoint, EP_DHCPData.ulLeaseTime );
+        vDHCP_RATimerReload( ( struct xNetworkEndPoint * ) pxEndPoint, EP_DHCPData.ulLeaseTime );
```

Comment:
Oops, FreeRTOS_DHCPv6.c also needs to include FreeRTOS_IP_Timers.h, possibly here:

~~~diff
     #include "FreeRTOS_Sockets.h"
     #include "FreeRTOS_IP_Private.h"
+    #include "FreeRTOS_IP_Timers.h"

     #include "FreeRTOS_BitConfig.h"
~~~

Reply:
Thanks for checking. Added at commit [68db748](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/537/commits/68db74848c67189b6a62c432603f2a70d00145c8).